management/server/account.go

```diff
@@ -50,6 +50,8 @@ const (
 	CacheExpirationMax         = 7 * 24 * 3600 * time.Second // 7 days
 	CacheExpirationMin         = 3 * 24 * 3600 * time.Second // 3 days
 	DefaultPeerLoginExpiration = 24 * time.Hour
+
+	DefaultPeerInactivityExpiration = 1 * time.Second
```

Comment:
Because of various reasons, a peer might be online while the management service is down. We should have a reasonable default to avoid issues with most folks who decide to enable this setting. 

As we have validated so far with ephemeral peers, a 10-minute timeout could be a good default and cover most cases.

We do have plans to add a logout option for the client when running `netbird down` or `netbird logout`